### PR TITLE
[huobi] error mapping

### PR DESCRIPTION
**Pull request by npomfret**

### CODE REVIEW

File: js/huobipro.js [hunk 1]
```diff
@@ -196,6 +196,10 @@ module.exports = class huobipro extends Exchange {
                 },
             },
             'exceptions': {
+                'board: {
```
Comment (kroitor) [inline js/huobipro.js:199]: Board = broad )

Comment (npomfret) [reply js/huobipro.js:199]: oops

**Comment by npomfret** [comment]
how do you edit a pr through GitHub?

**Comment by kroitor** [comment]
> how do you edit a pr through GitHub?

You open "files changed":

Screenshot 2021-02-19 at 20 47 35

And then click the ellipsis on the file:

**Comment by kroitor** [comment]
@npomfret you can also do the above from within your branch on github.

**Pull request merged**